plugins/jira/commands/create.md

```
@@ -1,14 +1,14 @@
 ---
 description: Create Jira issues (story, epic, feature, task, bug, feature-request) with proper formatting
-argument-hint: <type> [project-key] <summary> [--component <name>] [--version <version>] [--parent <key>]
+argument-hint: <type> [project-key] <summary> [--component <name>] [--version <version>] [--parent <key>] [--template <name>]
 ---
 
 ## Name
 jira:create
 
 ## Synopsis
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)